ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml

```
@@ -73,3 +73,4 @@
     - WiX                         # For creating installers
     - shortNames
     - Dragonwell                  # Dragonwell bootstrap image
+    - Thunderbird
```

I suggest adding the tags here rather than duplicating them on each task in the role

```suggestion
    - role: Thunderbird
      tags: [Thunderbird, jck]
```

---

I did not commit suggestion as I also had to remove the tags in the role's main.yml